gqueries/modules/multi_year_charts/flexible_capacity_chart/myc_flexible_capacity_in_energy_interconnector_exported.gql

```diff
@@ -0,0 +1,18 @@
+# The flexible capacity in energy interconnectors exported electricity
+
+- unit = MW
+- query =
+    SUM(
+        Q(energy_interconnector_1_exported_electricity_capacity),
```

Comment:
With an availability curve specified, it will disregard the availability specified in the file and just output the input_capacity. I tested this with an availability curve with only zero's and one with one's and zero's.